Makefile

```diff
@@ -67,8 +67,8 @@ docker-images:
 
 .PHONY: revendor
 revendor:
-	@GO111MODULE=on go mod vendor
```

Comment:
You could rebase. Similar change was already merged in https://github.com/gardener/gardener-extension-os-gardenlinux/commit/33fa566967eef6d38ca868c8526933d50e685653.